MaxText/pyconfig.py

```diff
@@ -482,7 +482,7 @@ def update_model_vars(base_config_path, raw_keys, config_name: str):
 
 def validate_megablox_parallelism(raw_keys):
   if raw_keys["megablox"] and (
-      using_sequence_parallelism(raw_keys) or using_pipeline_parallelism(raw_keys) or using_expert_parallelism(raw_keys)
+      using_sequence_parallelism(raw_keys) or using_expert_parallelism(raw_keys)
   ):
     raise ValueError("Currently we only support Megablox with data and tensor parallelism.")
```

Comment:
You may want to update the message in ValueError. Looking forward to see some baseline metrics! Thanks Matt!